apps/web-giddh/src/app/vat-report/obligations/obligations.component.html

```diff
@@ -98,8 +98,18 @@
                         <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
                     </table>
                     <section *ngIf="!tableDataSource?.length">
-                        <div class="no-data">
-                            <h1>{{ commonLocaleData?.app_no_data_found }}</h1>
+                        <div *ngIf="getFormControl('taxNumber').value">
+                            <div *ngIf="connectToHMRCUrl" class="initial-screen position-absolute text-center">
+                                <a mat-stroked-button color="primary" class="text-center" [href]="connectToHMRCUrl">
```

Comment:
use button tag

Reply:
Done

Reply:
connectToHMRCUrl has unique url , not redirect in giddh website.

Reply:
connectToHMRCUrl has unique url , not redirect in giddh website.